authentication.go

```diff
@@ -87,3 +88,24 @@ func (a *App) ValidateAuthToken(authToken string) (string, bool) {
 
 	return userID, true
 }
+
+// ValidateJwt determines whether a JWT is valid or not
+// returns userID (string) on success, error on failure
+func (a *App) ValidateJwt(authToken string) (string, bool) {
```

Comment:
since go prefers all caps acronyms
```suggestion
func (a *App) ValidateJWT(authToken string) (string, bool) {
```

Reply:
Made this change